packages/perseus/src/widgets/mock-widgets/mock-asset-loading-widget.tsx

```diff
@@ -18,7 +18,6 @@ export const mockedAssetItem: PerseusItem = {
                 static: false,
                 graded: true,
                 options: Object.freeze({value: ""}),
-                version: {major: 1, minor: 0},
```

Comment:
I think this was a mistake. This is a new widget, I don't think we've moved from 0 to 1 already. Also I don't think we need to upgrade a mock widget.

Reply:
Agreed. Looking at `widgets.ts` we have some [protective code](https://github.com/Khan/perseus/blob/7a984eba6f1cec3df314ec245d2176f5db190548/packages/perseus/src/widgets.ts#L239) that just skips if the data (v1.0) is _newer_ than the widget's version (v0.0).

But agree it's safe to remove this.